tests/common.go

```diff
@@ -4751,15 +4751,15 @@ func DeleteNfsSubPath(subPath string) {
 	creds := GetNfsInfoFromEnv()
 	mountDir := fmt.Sprintf("/tmp/nfsMount" + RandomString(4))
 
-	// Mount the NFS share to the master node.
-	masterNode := node.GetMasterNodes()[0]
```

Comment:
The error seen if master node doesn't have nfs utils. Hence changes the node to worker

{Failed to run command on: ip-10-13-194-139.pwx.purestorage.com. Cause: unable to run cmd (mount -t nfs nfs-pxbkp.pwx.dev.purestorage.com:/mnt/nfs-share-1 /tmp/nfsMounteyml): Failed to run command on: . Cause: failed to run command. sterr: mount: /tmp/nfsMounteyml: bad option; for several filesystems (e.g. nfs, cifs) you might need a /sbin/mount.<type> helper program.